{{cookiecutter.project_name}}/Makefile

```diff
@@ -121,6 +121,7 @@ README-pypi.html: README.rst
 	$(RST2HTML) README.rst README-pypi.html
 README.rst: README.md
 	pandoc -f markdown_github -t rst -o README.rst README.md
+	$(PYTHON) setup.py check -rsm
```

Comment:
I'd also like to switch the `check` options to their long form for better self-documentation.